frontend/src/v5/ui/controls/avatar/avatar.styles.ts

```diff
@@ -18,41 +18,41 @@
 import styled, { css } from 'styled-components';
 import { IconButton } from '@mui/material';
 
-export const StyledIconButton = styled(IconButton)<{ $isButton?: boolean, disabled?: boolean, $largeIcon?: boolean }>`
-	&& {
-		padding: 0;
-		margin: 0;
+export const StyledIconButton = styled(IconButton)<{ $isButton?: boolean, disabled?: boolean, $size?: number }>`
+	padding: 0;
+	margin: 8px 7px;
 
-		${({ $isButton, theme }) => ($isButton ? `
-			cursor: pointer;
-			
-			&:hover {
-				.MuiAvatar-root {
-					background-color: ${theme.palette.tertiary.mid};
-				}
+	${({ $isButton, theme }) => ($isButton ? `
+		cursor: pointer;
+		
+		&:hover {
+			.MuiAvatar-root {
+				background-color: ${theme.palette.tertiary.mid};
 			}
+		}
 
-			&:active {
-				.MuiAvatar-root {
-					background-color: ${theme.palette.tertiary.main};
-				}
+		&:active {
+			.MuiAvatar-root {
+				background-color: ${theme.palette.tertiary.main};
 			}
-		` : `
-			cursor: default;
-		`)}
+		}
+	` : 'cursor: default;')}
+
+	&:hover {
+		background-color: transparent;
 	}
 
-	.MuiAvatar-circle {
+	.MuiAvatar-circular {
 		margin: 0;
-		${({ $largeIcon }) => `
-			height: ${$largeIcon ? '48px' : '38px'};
-			width: ${$largeIcon ? '48px' : '38px'};
+		${({ $size }) => `
+			height: ${$size || 38}px;
+			width: ${$size || 38}px;
```

Comment:
I would prefer to have standardized sizes (unless there is a good reason).  It promotes reusability. Small, Medium, Large wouldnt cut it?

Reply:
Avatar is now accepting "large" | "medium" | "small", the latter is also the default value for it